DuckDuckGo/AppTPManageTrackersViewModel.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  AppTPManageTrackersViewModel.swift
+//  DuckDuckGo
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import Core
+
+struct ManagedTrackerRepresentable: Hashable {
+    let domain: String
+    let trackerOwner: String
+    let blocking: Bool
+}
+
+class AppTPManageTrackersViewModel: ObservableObject {
```

Comment:
I want to unit test this view model but since this PR is already huge I will do a follow up and refactor `TrackerDataParser` to be testable as well.